lib/core.js

```diff
@@ -2,6 +2,8 @@
 
 // Impress Application Server Core
 
+const APPLICATION_TEST_TIMEOUT = 5 * 60 * 1000; // 5m
```

Comment:
See below, all constants usually added to namespace `impress`, for example: `impress.DEFAULT_KEEP_ALIVE` so we need to move this constants to namespace or move all other constants to module scope.

Reply:
Done.